//
// Copyright 2019 Asylo authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto2";

package asylo;

// An enum to use in ProtoEnumValueName() tests.
enum TestProtoEnum {
  TEST_PROTO_ENUM_UNKNOWN = 0;
  TEST_PROTO_ENUM_VALUE_A = 1;
  TEST_PROTO_ENUM_VALUE_B = 1000;
  TEST_PROTO_ENUM_VALUE_C = -70;
}
